bot/seasons/valentines/be_my_valentine.py

```diff
@@ -0,0 +1,237 @@
+
+
+    **This command should be DMed to the bot.**
+
+
+    syntax : .bemyvalentine dm [user](optional) [p/poem/c/compliment/or you can type your own valentine message]
+    (optional)
+
+    example : .bemyvalentine dm (sends valentine as a poem or a compliment to a random user in DM making you anonymous)
+
+    example : .bemyvalentine dm Iceman#6508 p (sends a poem to Iceman in DM making you anonymous)
+
+    example : .bemyvalentine dm Iceman#6508 Hey I love you, wanna hang around ? (sends the custom message to Iceman in
+    DM making you anonymous)
+        """
+        if ctx.guild is not None:
+            # This command is only DM specific
+            msg = "You are not supposed to use this command in the server, DM the command to the bot."
+            return await ctx.send(msg)
+
+        if user == ctx.author:
+            # Well a user cant valentine himself/herself.
+            await ctx.send('Come on dude, you cant send a valentine to yourself :expressionless:')
+
+        elif user is None:
+            # just making sure that the random dosent pick up the same user(ctx.author)
+            USER_LOVEFEST.remove(ctx.author)
+            user = random.choice(USER_LOVEFEST)
+            USER_LOVEFEST.append(ctx.author)
+
+            if valentine_type is None:
+                valentine, title = self.random_valentine()
+                embed = discord.Embed(
+                    title=f"{EMOJI_1} {title} {user.display_name}{EMOJI_2}",
+                    color=Colours.pink
+                )
+
+            elif valentine_type in ['p', 'poem']:
+                valentine = self.valentine_poem()
+                embed = discord.Embed(
+                    title=f"{EMOJI_1} A poem dedicated to {user.display_name}{EMOJI_2}",
+                    color=Colours.pink
+                )
+
+            elif valentine_type in ['c', 'compliment']:
+                valentine = self.valentine_compliment()
+                embed = discord.Embed(
+                    title=f"{EMOJI_1} A compliment for {user.display_name}{EMOJI_1}",
+                    color=Colours.pink
+                )
+            else:
+                # in this case, the user decides to type his own valentine.
+                valentine = valentine_type
+                embed = discord.Embed(
+                    title=f'{EMOJI_1}A message for {user.display_name}{EMOJI_2}',
+                    color=Colours.pink
+                )
+
+            embed.description = f'{valentine} \n **{EMOJI_2}From anonymous{EMOJI_1}**'
+            await ctx.author.send(f"Your message has been sent to {user}")
+            await user.send(embed=embed)
+
+    @staticmethod
+    def random_valentine():
+        """
+            grabs a random poem or a compliment (any message)
+        """
+        with open(Path('bot', 'resources', 'valentines', 'bemyvalentine_valentines.json'), 'r', encoding="utf8") as f:
```

Comment:
Can we load the json data into a dict when the extension is loaded, so the file is only opened once, instead of on every command use. It'll speed things up in sacrifice of using some memory for storing it.